src/v/security/gssapi_authenticator.cc

```diff
@@ -176,6 +175,15 @@ gssapi_authenticator::gssapi_authenticator(
 gssapi_authenticator::~gssapi_authenticator() = default;
 
 ss::future<result<bytes>> gssapi_authenticator::authenticate(bytes auth_bytes) {
+    if (!_impl) {
+        vlog(
+          seclog.warn,
+          "authenticate received after handshake complete {} bytes",
+          _state,
+          auth_bytes.size());
```

Comment:
mismatch in fmt arguments and fmt placeholders.

Reply:
Fixed